docs/contributing.md

```diff
@@ -0,0 +1,202 @@
+
+After committing your changes, you'll be brought back to a pull request view comparing your new branch to `main`. You may need to edit the PR title, but if your files changed look good, click `Create pull request`, and you're done!\*
+
+\*Someone else will need to approve your PR (or give you feedback before approving), and then it will be merged and deployed by GH Pages.
+
+## Create a New Document
+
+> 👉 [Do this in the GitHub UI](https://github.com/mozilla/experimenter-docs/new/main/docs) 👈
+
+Let's say you want to create a new document called "My Flying Nimbus" that details how to acquire and guide your very own magical, yellow cloud.
+
+In the [`docs/`](https://github.com/mozilla/experimenter-docs/tree/main/docs) directory, either locally on your machine or through the GitHub UI link above, create your file, `my-flying-nimbus.md`. The name can be whatever you want, but it's a good general rule of thumb to keep the name similar to whatever the doc title will be.
+
+Paste this at the top of the file:
+
+```md
+---
+id: my-flying-nimbus
+title: My Flying Nimbus
+slug: /my-flying-nimbus
+---
+```
+
+With this at the top of the MD file, Docusaurus will automatically make your document available for viewing at the slug `/my-flying-nimbus`. Write your document in MD/X and once you're satisfied with your new document, commit and push your changes to your branch or follow the instructions in the [GitHub UI](#Github-UI) section to commit your changes and put in your pull request.
+
+## Modify an Existing Document
+
+> 👉 [Do this in the GitHub UI](https://github.com/mozilla/experimenter-docs/tree/main/docs) 👈
+
+Editing an existing doc is easy peasy. Click on the link above and choose the document you want to modify, or alternatively, find the "Edit this page" link at the bottom of any document inside the Doc Hub, and follow the instructions in the [GitHub UI](#Github-UI) section.
+
+Modifying the `title` property at the top of the document will update what the sidebar link displays.
+
+Note that if you change the ID also found at the top of the document, you will also need to update the matching string in `sidebars.js`. See the ["Adding or Modifying Sidebar Links"](#adding-or-modifying-sidebar-links) section for more details as well as [Doc Edits + Sidebar Edits](#doc-edits--sidebar-edits) in the GH UI.
+
+## Add or Modify Sidebar Links
+
+> 👉 [Do this in the GitHub UI](https://github.com/mozilla/experimenter-docs/blob/main/sidebars.js) 👈
+
+If you'd like to add a new external link, link to a new document, or modify an existing link in the sidebar, you have a few options.
+
+Note that if you'd like more details than what's provided here, you can check out the [Docusaurus Docs on sidebars](https://v2.docusaurus.io/docs/sidebar).
+
+1. Create a new top-level internal link to the doc or modify an existing one
+
+<img src="../img/top-level-link.png" alt="top-level link" style={{ width: '500px' }} />
```

Comment:
Oooh, @jodyheavener pointed out to me these aren't mobile-friendly, d'oh... I'll fix this up later today.